dotnet/HtmlFormatter/JsonInHtmlWriter.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HtmlFormatter;
```

Comment:
We should use the same structure for the namespace as we use for the package, so this should be `Cucumber.HtmlFormatter` (same for all other files in the project)

Reply:
Done